### DISCUSSION TITLE
Is there a way to see all raw packets?

### DISCUSSION THREAD

**Discussion by ddirector51** [merbanan/rtl_433#3004] [Q&A]
I have a smart outlet remote control.  I don't care much about decoding the packet; I just want to see when the button is pressed.  So, is there a way to get RTL_433 to show all received raw packets? This remote should be on the default frequency.

**Comment by merbanan** [comment]
The short answer is no. It does not work like that. But you can try running rtl_433 -A. This will invoke the built in analyzer maybe that is good enough for you.

**Comment by zuckschwerdt** [comment]
If you really want  to see all rf data regardless of decoding then either try `rtl_433 -vvvv` or `rtl_433 -w OOK:-` but as already answered this volume of data won't be useful.

As suggested try `-A ` or grab a packet, analyze the timings offline, s.a. https://triq.org/rtl_433/ANALYZE.html

Then write a flex decoder, roughly as suggested by `-A` and as see in https://github.com/merbanan/rtl_433/tree/master/conf
